ui/components/requirements/requirement.jsx

```diff
@@ -17,6 +17,19 @@ export default function Requirement({ requirement }) {
           <div className="sunset-date">
             Sunset date by { requirement.policy.sunset || 'none' }
           </div>
+          <div className="topics">
+            <ul className="topics-list list-reset">
+              <li className="inline">Topics: </li>
+              { requirement.keywords.map((keyword, index, keywords) => (
+                <li className="inline">
+                  { keyword }
+                  { index !== keywords.length - 1 &&
+                  <span>, </span>
+                  }
```

Comment:
This threw me for a loop until I saw it in action. What do you think about adding the commas in css instead? That'd simplify this logic a lot as we wouldn't need to track the index at all.

Unrelated: does the `li` need to have a key?